pyretis/meta.yaml

```diff
@@ -0,0 +1,60 @@
+#{% set name = "pyretis" %}
+#{% set version = "0.9.2" %}
+#{% set sha256 = "f738210a323dd8f84f98f33f0836dfdcc04f3fa8559a4e847f8eb311f147c7cc" %}
+
+package:
+   name: pyretis
+   version: 0.9.2
+#  name: {{ name|lower }}
+#  version: {{ version }}
+
+source:
+#   git_url: https://github.com/EnricoRiccardi/pyretis/tree/master/pyretisi
+#   git_url: https://gitlab.com/pyretis/pyretis
+   git_url: https://gitlab.com/pyretis/pyretis
+#  git_url: https://github.com/EnricoRiccardi/PyRETIS-on-GIT/tree/master/pyretis
+#  git_url: https://github.com/EnricoRiccardi/pyretis/edit/master/recipes/pyretis/pyretis
+
+  sha256: {{ sha256 }}
+
+build:
+#  preserve_egg_dir: True
+#  number: 0
+#  skip: True  # [ not py2k or win #]
+  script: python setup.py install 
+
+requirements:
+  build:
+    - python
+    - future
+    - setuptools
+    - numpy
+    - scipy
+    - pandas
+#    - mdtraj 
+    - matplotlib
+    - jinja2
+    - tqdm
+    - colorama
+
+  run:
+    - python
+    - future
+    - setuptools
+    - numpy
+    - scipy
+    - pandas
+#    - mdtraj
+    - matplotlib
+    - jinja2
+    - tqdm
+    - colorama
+
+test:
+  imports:
+    - pyretis
+
+about:
+  home: http://www.pyretis.org
+  license: LGPL 2.1 or later
+summary: ''
```

Comment:
You definitely need to add yourself as a maintainer. See:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L67-L72